applications/asset_tracker_v2/src/cloud/cloud_codec/aws_iot/aws_iot_codec.c

```diff
@@ -62,7 +62,7 @@ int cloud_codec_encode_cloud_location(
 	struct cloud_codec_data *output,
 	struct cloud_data_cloud_location *cloud_location)
 {
-	int err;
+	int err = -ENODATA;
```

Comment:
Because this function is not called when there are no neighbor cells or WiFi access points, I had removed the branch returning `-ENODATA`. I'm thinking if we should add it back as static analysis tool seem to complain about it. The main question is whether `cJSON_PrintUnformatted()` works with "empty" cJSON object. I'd expect it does but maybe the function is a little vulnerable after I removed `-ENODATA` branch.

Reply:
I see. Would you recommend getting this in and then reworking the code in a later point of time?

Reply:
Yep, you can get this merged.